src/app/components/FormInput/FormInput.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import styled from 'styled-components'
+
+type FormInputProps = {
+  type: 'text' | 'textArea'
+  value: string
+  name: string
+  diff?: number
+  onChange: (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => void
+  required?: boolean
+}
+
+export default function FormInput({
+  type,
+  value,
+  onChange,
+  name,
+  diff,
+  required,
+}: FormInputProps): JSX.Element {
+  return (
+    <InputWrapper>
+      <FormLabel htmlFor={name}>{name}</FormLabel>
+      {type === 'text' ? (
+        <Input
+          type="text"
+          name={name}
+          required={required}
+          value={value}
+          onChange={onChange}
+        />
+      ) : (
+        <TextArea
+          name={name}
+          rows={10}
+          value={value}
+          onChange={onChange}
+          required={required}
+        />
+      )}
+      {diff && <TextLimitDisplay diff={diff}>{`${diff}`}</TextLimitDisplay>}
+    </InputWrapper>
+  )
+}
+
+const FormLabel = styled.label`
+  display: block;
+  font-weight: bold;
+  text-transform: capitalize;
+`
+const Input = styled.input`
+  width: 100%;
+  background-color: transparent;
+  border: 1px solid rgba(0, 0, 0, 0.5);
+  padding: 8px 12px;
+  margin-bottom: 20px;
+  border-radius: 12px;
+  font-weight: bold;
+  font-size: 1.125rem;
```

Comment:
Why using rem here but px before and after?
```suggestion
  font-size: 18px;
```